### PR TITLE
ref: Consolidate QR codes

### PR DESCRIPTION
**Pull request by matejminar**
Here(https://github.com/getsentry/sentry/pull/23981) we introduced a new frontend package for QR codes.

In this PR I removed the old QR code component that worked with matrix data.
I refactored the backend because there's no need to send matrix data anymore - the new package is able to generate QR codes from strings.

### CODE REVIEW

File: src/sentry/utils/otp.py [hunk 1]
```diff
@@ -1,7 +1,6 @@
 import time
 import hmac
 import base64
-import qrcode
```
Comment (billyvg) [inline src/sentry/utils/otp.py:4]: Is this a deep that can be removed now?

**Review by billyvg — APPROVED** [review]

File: src/sentry/utils/otp.py (hunk 1 above)
Comment (matejminar) [reply src/sentry/utils/otp.py:4]: I hope so, wanted to check with backend folks first

Comment (EvanPurkhiser) [reply src/sentry/utils/otp.py:4]: Looks reasonable to remove to me

**Review by evanpurkhiser — APPROVED** [review]
Really good cleanup

File: src/sentry/utils/otp.py (hunk 1 above)
Comment (markstory) [reply src/sentry/utils/otp.py:4]: Can it also be removed from the getsentry dependencies as well?

**Review by untitaker — APPROVED** [review]
only approving requirements-base

File: src/sentry/utils/otp.py (hunk 1 above)
Comment (matejminar) [reply src/sentry/utils/otp.py:4]: It's not being used there. Here's PR: https://github.com/getsentry/getsentry/pull/5184

**Pull request merged**